README.md

```diff
@@ -71,7 +76,9 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+friends.each do |friend|
+  puts "Belle is friends with " + friend
```

Comment:
String interpolation would be better here in terms of style, and because it auto-converts the value to a string in case it's not:

```ruby
puts "Belle is friends with #{friend}"
```